src/action.ts

```diff
@@ -5,6 +5,8 @@ import * as versionbot from './versionbot-utils';
 import * as balena from './balena-utils';
 import * as git from './git';
 
+import { createRef } from './github-utils';
```

Comment:
Group the local imports

```suggestion
import * as git from './git';
import { createRef } from './github-utils';
```

Reply:
Sure thing!